components/Footer/Footer.module.css

```diff
@@ -1,69 +1,54 @@
 .footer {
     background-color: var(--dark);
-    padding: 2rem;
+    padding: 16px;
+    padding-bottom: 32px;
     color: #fff;
     margin-top: 2rem;
 }
-.footer__top {
-    display: flex;
-    justify-content: space-around;    
-    flex-wrap: wrap;
+.footerTop {
+    padding-bottom: 16px;
     border-bottom: 1px solid var(--lightGrey);
-    padding: 2rem;
-}
-.footer__linkHeading {
```

Comment:
Uh, I never liked BEM, this mix of `_` and `-`.
In this case, it was a BEM/came case mix!